crt_portal/cts_forms/templatetags/back_to_all.py

```diff
@@ -14,6 +14,5 @@ def back_to_all(return_url_args=''):
         return return_url_args
     violation_summary = querydict.get('violation_summary', '')
     if violation_summary.startswith('^') and violation_summary.endswith('$'):
-        querydict.pop('violation_summary')
```

Comment:
confirming: we don't need to pop violation_summary from querydict?

Reply:
no because I remove it below on line 17